packages/cache-browser-local-storage/src/createBrowserLocalStorageCache.ts

```diff
@@ -64,10 +67,14 @@ export function createBrowserLocalStorageCache(options: BrowserLocalStorageOptio
       });
     },
 
-    clear(): Readonly<Promise<void>> {
+    clear() {
       return Promise.resolve().then(() => {
         getStorage().removeItem(namespaceKey);
       });
     },
+
+    entries() {
```

Comment:
Who's calling `entries()` ?

Reply:
this method isn't strictly necessary, but this makes the format of the cache opaque in case you want to transfer from localStorage to another place. In this case it's mainly there for consistency with the memory cache.

In theory the local storage cache could also accept a entries as initialCache, like memory cache does now.

If we don't have an entries value, we have to rely on the format not changing between versions